peercrawler.py

```diff
@@ -295,7 +295,7 @@ def get_label(p: Peer) -> str:
         graph = Dot("network_connections", graph_type="digraph")
         for node, peers in self.get_connections_graph().items():
             for peer in peers:
-                if only_voting and not (node.is_voting and peer.is_voting):
+                if filter_function is not None and not (filter_function(node) and filter_function(peer)):
```

Comment:
What do you think about calling the filter function  `should_keep`?
It makes it easier to read the code